### ISSUE TITLE
Request batching doesn't work when library is initialized after dom load

### ISSUE THREAD

**Issue by SuperKruse** [PostHog/posthog-js#66]
Hey,

When using posthog-js as a module there can be a problem with starting the _event_queue_poll.

I think the problem is when the library is initialized after the dom is loaded.

Looks like `dom_loaded_handler` runs on all posthog instances when the dom is loaded, but if there ins't any instances yet, they aren't loaded.

A quick fix suggestion is to add `this._dom_loaded()` to `_loaded`. 

```javascript
PostHogLib.prototype._loaded = function() {
    this.get_config('loaded')(this);

    this._dom_loaded()

    // this happens after so a user can call identify in
    // the loaded callback
    if (this.get_config('capture_pageview')) {
        this.capture_pageview();
    }
};
```
And add a clearInterval to `_event_queue_poll` to make sure there is only one instance of the timer.

```javascript
PostHogLib.prototype._event_queue_poll = function () {

    clearInterval(this._poller);
    
    const POLL_INTERVAL = 3000;
    this._poller = setTimeout(() => {  
      ...

```

A workaround is to set `request_batching: false` when initializing posthog.

**Comment by mariusandra** [comment]
Hey! I released `1.3.2` with a fix for this. Please let me know if that fixes the issue! :)

**Comment by SuperKruse** [comment]
Thanks, it works now!
